languages/tree-sitter-stack-graphs-javascript/src/stack-graphs.tsg

```diff
@@ -2412,6 +2412,9 @@ inherit .return_or_yield
 
   ; the value of undefined is the undefined primitive
   edge @undefined.value -> @undefined.builtins_undefined
+
+  ; HACK: `undefined` is a perfectly cromulent name for a parameter, but the parser thinks it means this node instead. For the moment, work around the problem this causes by giving all `undefined` nodes covalues like parameters enjoy.
```

Comment:
My only suggestion is add `!!!!` before the `HACK` to be in line with other similar comments. I try to use four exclamation points for things that we'll want to search for later, and then use a tag like "TODO" or "HACK" to distinguish what sort they are.

Reply:
Good call 👍🏻